src/libraries/Actions.sol

```diff
@@ -32,4 +32,8 @@ library Actions {
     // mint + burn ERC721 position
     uint256 constant MINT_POSITION = 0x22;
```

Comment:
thoughts on moving these to the pools action section? since they also modify liquidity